compiler/rustc_hir_analysis/src/outlives/implicit_infer.rs

```diff
@@ -196,6 +196,7 @@ fn insert_required_predicates_to_be_wf<'tcx>(
                 }
             }
 
+            // FIXME(fmease): Do we need to handle inherent projections similarly here?
```

Comment:
Probably? it would be nice to create a test exercising this behavior... maybe something like:

```
struct Foo;

impl Foo {
  type IAT<'a, T> = &'a T;
}

struct Bar<'a, T>(Foo::IAT<'a, T>);
```

Unless I'm getting this mixed up with other implicit outlives code :upside_down_face:

Reply:
Right now I have a negative behavior test for `type IAT<'a, T> = &'a T;` (i.e. no implied bounds).